src/triton_controls/config/thruster_config.yaml

```diff
@@ -6,41 +6,31 @@
       contrib: 
         x: -0.7071
         y: 0.7071
-      lx: -0.235
-      # lx: 6.5640
-      ly: -0.189 
-      # ly: 6.5640
+      lx: -0.5
```

Comment:
It looks like these were changed for testing with the cube model, should we put these in a separate config or revert them?